website/docs/docs/cloud/manage-access/auth0-migration.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Migrating to Auth0 for SSO"
+id: "auth0-migration"
+sidebar: "SSO Auth0 Migration"
+description: "Required actions for migrating to Auth0 for SSO services on dbt Cloud"
+---
+
+dbt Labs is partnering with Auth0 to bring enhanced features to dbt Cloud's single sign-on (SSO) capabilities. Auth0 is an identity and access management (IAM) platform with advanced security features, and it will be the dbt Cloud SSO identity provider (IDP) moving forward. These changes will require some action from customers with SSO configured in dbt Cloud today, and this guide will outline the necessary changes for each environment. 
+
+If you have not yet configured SSO in dbt Cloud, refer instead to our setup guides for [SAML](docs/cloud/manage-access/set-up-sso-saml-2.0), [Okta](docs/cloud/manage-access/set-up-sso-okta), [Google Workspace](docs/cloud/manage-access/set-up-sso-google-workspace), or [Azure Active Directory](docs/cloud/manage-access/set-up-sso-azure-active-directory) single sign-on services.
+
+## SAML 2.0 and Okta
+
+SAML 2.0 users must update a few fields in the SSO app configuration to match the new Auth0 URL and URI.  You can approach this by editing the existing SSO app settings or creating a new one to accommodate the Auth0 settings. One approach isn't inherently better, so choose whichever works best for your organization.
+
+The fields that will be updated are:
+- Single sign-on URL &mdash; `https://us-devspace-cluster.us.auth0.com/login/callback?connection={slug}`
+- Audience URI (SP Entity ID) &mdash; `urn:auth0:us-devspace-cluster:{slug}`
```

Comment:
@matthewshaver We could use the pattern we use today. Using a placeholder that points to a table for the different Auth0 tenants instead of hard-coding `us-devspace-cluster.us.auth0.com`. Custom domains for prod is coming which will make these urls look like [this](https://www.notion.so/dbtlabs/Auth0-tenants-and-custom-domains-7e8c90aee9d541e8b69bc5f3eb3e6206?pvs=4#af17f595f2aa44e2906b810d3f275b61)

https://docs.getdbt.com/docs/cloud/manage-access/set-up-sso-saml-2.0

cc @robotmiller @thorn14 @saimaddali

Reply:
agree with Ajay on this that it would be best to follow the existing pattern. [this PR](https://github.com/dbt-labs/docs.getdbt.com/pull/2933) I opened a while ago updates the SSO setup docs on the `next` version of the docs site, and something similar could be followed here I think.